src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -90,14 +90,8 @@ private static IEnumerable<string> GetValidAudiences()
         {
             if (SystemEnvironment.Instance.IsPlaceholderModeEnabled())
             {
-                if (SystemEnvironment.Instance.IsLinuxConsumptionOnLegion())
```

Comment:
The old LinuxConsumptionOnLegion helper was intended for Flex, but we don't run in-proc on Flex. Most of the Flex stuff was removed in https://github.com/Azure/azure-functions-host/pull/10213, but some helpers remain.